apps/web/models.py

```diff
@@ -48,10 +48,12 @@ class Team(models.Model):
         upload_to="bg-images", null=True, blank=True
     )
     team_type = models.CharField(choices=TEAM_TYPE_OPTIONS, max_length=50)
+    position = models.PositiveIntegerField(default=0)
 
     def __str__(self):
         return self.name
 
     class Meta:
         app_label = "web"
         db_table = "teams"
+        unique_together = (("position", "team_type"),)
```

Comment:
This will fail since you have setup default as 0. You will have to skip this check or write a script that assigns values to all the existing entries in the database.

Reply:
yeah..this will conflict with the existing data in the database.